new/INS1021Marenadaran.xml

```diff
@@ -5,8 +5,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title>Mesrop Mashtots Research Institute of Ancient Manuscripts
+                <title xml:lang="en">Mesrop Mashtots Research Institute of Ancient Manuscripts
```

Comment:
the different names are not the titles of the file (titleStmt) but the variants of the placeName and must be moved there
(placeName is treated in the same way as the work title or persName with n="1" n="2" and/or `corresp`)
https://betamasaheft.eu/Guidelines/?id=places

Reply:
Thank you! Can you also explain me why I can´t see anything in Armenian in Oxygen, but I see it in Gitup history and what should I change to enable the reading?

Reply:
You probably have to switch the main font to a Unicode font that has Armenian, I do not know which one you have, I use TITUS, it has Armenian. Ethiopic Unicode does not have Armenian or most other Oriental fonts.

Reply:
![image](https://github.com/user-attachments/assets/c1164185-c1cc-4044-bc09-4c95545f8fbc)

Reply:
Thank you! I am trying to download TITUS but it seems not to be available. If you can direct me to the right page or send me the font via email, I would be very grateful!